Vagrantfile

```diff
@@ -0,0 +1,36 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+ENV["TERM"] = "xterm-256color"
+ENV["GOPATH"] = "/go"
+ENV["LC_ALL"] = "en_US.UTF-8"
+
+Vagrant.configure("2") do |config|
+  # config.vm.box = "fedora/25-cloud-base"
+  # config.vm.provision "shell", inline: "dnf install -y go git docker"
+  config.vm.box = "ubuntu/zesty64"
+  config.vm.provision "shell", inline: "DEBIAN_FRONTEND=noninteractive apt-get install -y golang git docker.io systemd-container tmux"
+
+  # config.vbguest.auto_update = true
+
+  config.vm.synced_folder ".", "/vagrant", disabled: true
+  config.vm.synced_folder ".", "/go/src/github.com/kinvolk/kubeadm-nspawn", create: true
```

Comment:
I have not tried `make vendor` before.

When trying:
```
$ make vendor
package github.com/Masterminds/glide: mkdir /go/src/github.com/Masterminds/: permission denied
Makefile:15: recipe for target 'glide' failed
make: *** [glide] Error 1
```
What about preparing the GOPATH in `/home/ubuntu/go` instead of `/go`, and then make sure it belongs to the correct user?